src/api/APIManager.ts

```diff
@@ -14,13 +18,12 @@ export class APIManager {
 	 * @param query
 	 * @param apisToQuery
 	 */
-	async query(query: string, apisToQuery: string[]): Promise<MediaTypeModel[]> {
+	async query(query: string, apisToQuery: string[], mediaType: string): Promise<MediaTypeModel[]> {
```

Comment:
It would probably be better if you did the filtering of which APIs to query (based on the user's settings) before calling this method and leave the method as it was.